android/app/src/main/java/io/mosip/registration_client/api_services/BiometricsDetailsApi.java

```diff
@@ -500,6 +500,8 @@ public void getBioAttempt(@NonNull String fieldId, @NonNull String modality,
     public void startOperatorOnboarding(@NonNull BiometricsPigeon.Result<String> result) {
         try {
             userOnboardService.getOperatorBiometrics().clear();
+            userOnboardService.setIdaResponse(false);
+            userOnboardService.setIsOnboardSuccess(false);
```

Comment:
what is the utility of this method?

Reply:
This method resets the value of variables we use in the operator onboard process.